tensorrt_llm/_torch/autotuner.py

```diff
@@ -483,9 +484,10 @@ def _profile_single_kernel(self, runner: TunableRunner,
 
         avg_time = start.elapsed_time(end) / self.repeat
 
-        # Handle None tensors for optional inputs
+        # Handle None tensors for optional inputs and non-Tensor scalars
         shapes = [
-            t.size() if t is not None else torch.Size((0, )) for t in inputs
+            t.size() if isinstance(t, torch.Tensor) else torch.Size((0, ))
```

Comment:
NIT. 
t.size() if isinstance(t, torch.Tensor) else torch.Size((0, ))
is used multiple times, maybe considering make it as function.